src/org/opendatakit/briefcase/reused/http/RequestBuilder.java

```diff
@@ -220,9 +220,14 @@ public static boolean isUri(String candidate) {
   }
 
   public RequestBuilder<T> withPath(String path) {
-    if (!baseUrl.getPath().isEmpty())
-      throw new BriefcaseException("Can't apply withPath() twice");
-    URL newBaseUrl = url(baseUrl + (path.startsWith("/") ? path : "/" + path));
+    String cleanPath = path.startsWith("/") && path.endsWith("/")
+        ? path.substring(1, path.length() - 1)
+        : path.startsWith("/")
+        ? path.substring(1)
+        : path.endsWith("/")
+        ? path.substring(0, path.length() - 1)
+        : path;
+    URL newBaseUrl = url(baseUrl + "/" + cleanPath);
```

Comment:
Is this more readable?
<img width="1749" alt="Screen Shot 2019-06-30 at 11 44 38 AM" src="https://user-images.githubusercontent.com/55099/60400705-98723a80-9b2c-11e9-90c6-ff95c54e5018.png">

```
    int start = path.startsWith("/") ? 1 : 0;
    int end   = path.endsWith  ("/") ? 1 : 0;
    String cleanPath = path.substring(start, path.length() - end);
```

Reply:
startOffset and endOffset would be better

Reply:
Much better, thanks!